packages/repository/src/__tests__/unit/repositories/legacy-juggler-bridge.unit.ts

```diff
@@ -411,9 +414,26 @@ describe('DefaultCrudRepository', () => {
   });
 
   it(`throws error when execute() not implemented by ds connector`, async () => {
-    const repo = new DefaultCrudRepository(Note, ds);
+    const repo = new DefaultTransactionalRepository(Note, ds);
```

Comment:
I don't see why is this change necessary - how is `execute` related to transactions? Can you revert this line please?